src/style.css

```
@@ -0,0 +1,67 @@
+body {
+  margin: 0;
+}
+
+.star {
+  display: flex;
+  width: 96px;
+  justify-content: space-between;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 .stars__star:nth-child(1),
+.stars--2 .stars__star:nth-child(1),
+.stars--3 .stars__star:nth-child(1),
+.stars--4 .stars__star:nth-child(1),
+.stars--5 .stars__star:nth-child(1) {
+  width: 16px;
+  height: 16px;
+  background-image: url("images/star-active.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--2 .stars__star:nth-child(2),
+.stars--3 .stars__star:nth-child(2),
+.stars--4 .stars__star:nth-child(2),
+.stars--5 .stars__star:nth-child(2) {
+  width: 16px;
+  height: 16px;
+  background-image: url("images/star-active.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--3 .stars__star:nth-child(3),
+.stars--4 .stars__star:nth-child(3),
+.stars--5 .stars__star:nth-child(3) {
+  width: 16px;
+  height: 16px;
+  background-image: url("images/star-active.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--4 .stars__star:nth-child(4),
+.stars--5 .stars__star:nth-child(4) {
+  width: 16px;
+  height: 16px;
+  background-image: url("images/star-active.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--5 .stars__star:nth-child(5) {
+  width: 16px;
+  height: 16px;
+  background-image: url("images/star-active.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
```

```suggestion
 .stars--1 .stars__star:nth-child(-n+1),
 .stars--2 .stars__star:nth-child(-n+2),
.stars--3 .stars__star:nth-child(-n+3),
.stars--4 .stars__star:nth-child(-n+4),
.stars--5 .stars__star:nth-child(-n+5) {
  background-image: url("./images/star-active.svg");
}
```
it can be much shorter)
and here we need to change just `background-image`. other properties already settled in `.stars__star` selelctor and not will be changed.